charts/private-link/Chart.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: 0.1.0
+description: Synadia Private Link
+name: private-link
+version: 0.1.0
```

Let's start it off as an RC

```suggestion
version: 0.1.0-rc.0
```